site/styles/_meta-detail.scss

```diff
@@ -127,13 +127,13 @@
 
   @include media-breakpoint-up(lg) {
     flex-direction: row;
-    padding-bottom: 0;
     padding: var(--page-detail-padding-top-lg) 50px 0 50px;
+    padding-bottom: 0;
```

Comment:
Isn't this padding redundant? It's already being set on the line above.

Reply:
I think this is just something the linter moved lol removed